scripts/gen-docs.gpt

```diff
@@ -0,0 +1,14 @@
+tools: system-tools-state, sys.read, sys.write
```

Comment:
This is pretty cool :)

Reply:
I like it! It'll be cool if we can figure out some cost-effective way to run this in CI.